libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -261,7 +272,7 @@ - (SFSyncState*) reSync:(NSNumber*)syncId updateBlock:(SFSyncSyncManagerUpdateBl
         [self log:SFLogLevelError format:@"Cannot run reSync:%@:wrong type:%@", syncId, [SFSyncState syncTypeToString:sync.type]];
         return nil;
     }
-    [self logAnalyticsEventWithSyncState:sync name:@"reSync"];
```

Comment:
reSync is a way to re-run a previously created sync.
It will get only a subset of the target records if a maxTimeStamp is found, otherwise it is really just running the full sync.
Should we then simply add the sync id in the ailtn record?
We could then count distinct sync ids or the number of times a given sync is re-used etc.
In that case I could stop creating an event in syncUpWithOptions, syncDownWithOptions and reSync